spmd/compiler/graph_utils.py

```diff
@@ -41,12 +37,12 @@ def get_node_tensor_size(node: fx.Node, debug=False) -> int:
     return size
```

Comment:
Cannot review the accidentally committed line. So I comment here. Instead of manually calculating the size, you should be able to use `size = tdata.shape.numel()`.

Reply:
This is interacting with the node.meta dict, rather than an actual tensor though. 
Thus, we have a dict holding shape as a tuple, dtype, stride and memory format...thus have to manually calc as it's just two ints from a dict rather than a live tensor.